.pre-commit-hooks.yaml

```diff
@@ -1,5 +1,5 @@
 -   id: git-secrets
     name: Git Secrets
     description: git-secrets scans commits, commit messages, and --no-ff merges to prevent adding secrets into your git repositories.
-    entry: 'git-secrets --pre_commit_hook'
+    entry: pre-commit-hook-exec.sh
```

Comment:
Could we instead just do `entry: './git-secrets --pre-commit-hook'` (possibly even minus the quotes)?

I am not too familiar with [pre-commit](https://pre-commit.com/) so I understand I might be missing some context here.

That said, your solution may also be preferable if your use of `git secrets` causes existing git configuration to be honored in a way that directly calling the `git-secrets` script does not.